### ISSUE TITLE
Question about exporting PTv3 to ONNX Runtime with dynamic point counts

### ISSUE THREAD

**Issue by liufanfan1** [Pointcept/Pointcept#611]
Hi Pointcept team,

Thank you for releasing and maintaining this excellent project.

I am currently deploying a custom semantic segmentation model based on Point Transformer V3 on a Windows laptop with an NVIDIA GPU. The original PyTorch inference pipeline works correctly with FP16, FlashAttention, spconv, and variable-sized point-cloud fragments.

I am now investigating whether the model can be deployed using ONNX Runtime. I noticed the related discussions in:

- #143
- #492

My current experimental ONNX export uses a fixed number of points, for example `N = 4096`. However, real LAS point clouds produce fragments with different numbers of points.

Using a fixed input size creates two problems:

1. Fragments with fewer than 4096 points require padding. Naively padding or duplicating points may affect voxelization, sparse convolution, attention, and the final prediction.
2. Fragments with more than 4096 points require splitting or truncation, which may lose spatial context and reduce segmentation quality.

Ideally, I would like to export the model with a dynamic point dimension, for example:

```python
dynamic_axes = {
    "coord": {0: "num_points"},
    "grid_coord": {0: "num_points"},
    "feat": {0: "num_points"},
    "seg_logits": {0: "num_points"},
}

However, I understand that PTv3 depends on operations such as spconv, serialization, sorting/scatter operations, and custom CUDA kernels, which may not be directly supported by standard ONNX Runtime.
My questions are:
Is there currently a recommended way to export PTv3 to ONNX or ONNX Runtime with a dynamic number of points?
Has the latest Pointcept version added, or planned to add, an ONNX-friendly inference path?
If dynamic point counts are not supported, is padding with a validity mask a safe approach for PTv3? If so, which stages need to apply the mask?
Are custom ONNX Runtime operators or TensorRT plugins currently the only practical way to deploy PTv3/spconv outside PyTorch?
Would replacing spconv with an OCNN-based implementation make the model more exportable, or are there still unsupported PTv3 operations?

At the moment, the PyTorch/PyInstaller version works correctly, so this question is mainly about finding a more portable and efficient deployment path without changing the model’s predictions.
Any guidance, examples, experimental branches, or future plans would be greatly appreciated.
Thank you!
